src/main/java/com/bitso/BitsoBank.java

```diff
@@ -0,0 +1,20 @@
+package com.bitso;
+
+import org.json.JSONObject;
+
+import com.bitso.helpers.Helpers;
+
+public class BitsoBank {
```

Comment:
Consider if it would be more convenient for the response of getBitsoBanks() to be a HashMap, so that when you use it calling debit_card_withdrawal endpoint you can directly access the code for your bank without requiring an iteration over every bank.